packages/components/src/form-token-field/style.scss

```diff
@@ -46,10 +46,15 @@
 }
 
 .components-form-token-field__label {
-	display: inline-block;
+	display: block;
 	margin-bottom: $grid-size-small;
 }
 
+.components-form-token-field__help {
+	display: block;
```

Comment:
This applies to a `<div>` element, which is a block element by default. Unless I'm missing something, please remove this property. Also, I'd make this a `<p>` element, see comment above.